docs/dm/layers.md

```diff
@@ -30,8 +30,8 @@ Otherwise this layer behaves the same as the tokens layer.
 This layer cannot be manually selected, but it can be configured using the [DM Options Menu](/docs/dm/settings.md).
 This layer draws gridlines on the screen.
 
-Hexagonal grids are not currently supported.
-This is on my to-do list, but don't expect it immediately.
+Those gridlines are either squares (default) or hexagons.
+Both flat-top and point-top hex-grids can be drawn, yet are only experimentally supported, right now.
```

Comment:
Maybe mention that grid snapping does not properly work yet (i.e. it's still snapping based on a square grid, I should probably get to fix that or outright disable it when using hexes for now)